frontend/src/components/AccountReceiveTable.vue

```diff
@@ -97,7 +97,7 @@
               <q-card-section>
```

Comment:
Getting this on the Send page 
![image](https://user-images.githubusercontent.com/61768337/225680091-c2e609a3-1822-44c7-93cf-ec6a561fa65f.png)

Reply:
![image](https://user-images.githubusercontent.com/61768337/225690291-0cc9b925-9504-4386-8433-f3aba8ea96e5.png)

Reply:
I'm no longer encountering this issue